### DISCUSSION TITLE
Material Icons Issue...

### DISCUSSION THREAD

**Discussion by bigtopfamily** [squidfunk/mkdocs-material#5495] [Q&A] [answered]
Hi, I'm very new to mkdocs, and not technical, so please forgive the noob. I am using Material for Mkdocs for online help user guides for my organization. 

As our UI changes more and more, I realize that the icons between what you've recommended here  (https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/) which is located here (https://pictogrammers.com/library/mdi/) are not lining up with what my company uses here (https://fonts.google.com/). Like I said, I'm very new - be gentle - but is there a reason for this or a way to workaround it? Thanks!

**Comment by squidfunk** [comment]
Thanks for asking. Could you please clarify what you mean by "do not align"? Material for MkDocs ships the exact same version, that is 7.2.96, so I'm not sure what the problem is. Note that the Material Design icon set [is not the only one we ship](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#configuration).

https://github.com/squidfunk/mkdocs-material/blob/88515d26a1142a9f92f47af70008b1a7d6e9e818/package.json#L57

**Comment by bigtopfamily** [comment]
Thank you for replying! So there are discrepancies in the names of the icons – Google’s Material “sleep” icon looks like a moon and stars and the  “sleep” icon at the Pictogrammers material design icon is a series of “zzzz’s.”.

So, when I use the :material-sleep: and then serve up my help documentation, the icons that I’m using do not align with the ones reflected in our company’s applications. Which of course causes confusion.

There are many disparities like this – sleep is just an example. I’m not sure why or if there is a workaround? Thanks for your insight!




Ashley Allen
Technical Writer | Knowmadics, Inc
703-861-4137


From: Martin Donath ***@***.***>
Date: Wednesday, May 10, 2023 at 3:27 AM
To: squidfunk/mkdocs-material ***@***.***>
Cc: Ashley Allen ***@***.***>, Author ***@***.***>
Subject: Re: [squidfunk/mkdocs-material] Material Icons Issue... (Discussion #5495)

Thanks for asking. Could you please clarify what you mean by "do not align"? Material for MkDocs ships the exact same version, that is 7.2.96, so I'm not sure what the problem is. Note that the Material Design icon set is not the only one we ship<https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#configuration>.

https://github.com/squidfunk/mkdocs-material/blob/88515d26a1142a9f92f47af70008b1a7d6e9e818/package.json#L57

**Reply by squidfunk** [reply, marked as answer]
This is not an issue with Material for MkDocs. You can raise this to the maintainers of the Material Design icons, but since this is an upstream dependency of ours, we cannot change the icon shortcodes.

**Comment by bigtopfamily** [comment]
OK – thank you! Appreciate the feedback.

Ashley Allen
Technical Writer | Knowmadics, Inc
703-861-4137


From: Martin Donath ***@***.***>
Date: Wednesday, May 10, 2023 at 12:06 PM
To: squidfunk/mkdocs-material ***@***.***>
Cc: Ashley Allen ***@***.***>, Author ***@***.***>
Subject: Re: [squidfunk/mkdocs-material] Material Icons Issue... (Discussion #5495)

This is not an issue with Material for MkDocs. You can raise this to the maintainers of the Material Design icons, but since this is an upstream dependency of ours, we cannot change the icon shortcodes.
